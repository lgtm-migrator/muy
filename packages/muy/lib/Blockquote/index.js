import React from "react"
import Box from "@material-ui/core/Box"
import makeStyles from "@material-ui/core/styles/makeStyles"

const useStyles = makeStyles(theme => ({
  root: {
    "& *:last-child": {
      marginBottom: 0,
    },
  },
}))

const Blockquote = props => {
  const classes = useStyles()
  return (
    <Box
      className={classes.root}
      component="blockquote"
      mb={3}
      mr={3}
      mt={0}
      p={0}
      {...props}
    />
  )
}

export default Blockquote
