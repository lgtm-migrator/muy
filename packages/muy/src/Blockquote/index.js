import React from "react"
import makeStyles from "@material-ui/core/styles/makeStyles"

const useStyles = makeStyles(theme => ({
    root: {
        margin: `0 ${theme.spacing(3)}px ${theme.spacing(3)}px`,
        padding: 0,
        "& *:last-child": {
            marginBottom: 0,
        },
    },
}))

const Blockquote = props => {
    const classes = useStyles()
    return <blockquote className={classes.root} {...props} />
}

export default Blockquote
