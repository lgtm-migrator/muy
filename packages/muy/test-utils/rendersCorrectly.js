import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
