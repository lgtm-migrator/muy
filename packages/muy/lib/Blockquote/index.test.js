import React from "react";
import Blockquote from "../../lib/Blockquote";
import rendersCorrectly from "../../test-utils/rendersCorrectly";

describe('<Blockquote />', () => {
    it("renders correctly", () => {
        rendersCorrectly(<Blockquote/>)
    })
})
