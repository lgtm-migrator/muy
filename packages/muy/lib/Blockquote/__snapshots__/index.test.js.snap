// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blockquote /> renders correctly 1`] = `
<blockquote
  className="MuiBox-root MuiBox-root-3 makeStyles-root-1"
/>
`;
